spot_rl_experiments/spot_rl/envs/skill_manager.py

```diff
@@ -55,14 +58,14 @@ def __init_spot(self):
 
         # Construct configs for nav, gaze, and place
         self.nav_config = construct_config_for_nav()
-        self.pick_config = construct_config_for_gaze(
-            max_episode_steps=350
-        )  # TODO: Find a better episode cap
+        self.pick_config = construct_config_for_gaze(max_episode_steps=500)
```

Comment:
@jimmytyyang 500 is the default episode length limit inside the config, no point in manually feeding it 500.

We can shrink it to 350 like Sergio did at CVPR.

Reply:
Thanks @jimmytyyang  for addressing this.